src/libs/actions/Report.ts

```diff
@@ -1928,10 +1928,13 @@ function updateRoomVisibility(reportID: string, previousValue: RoomVisibility |
  * @param parentReportID The reportID of the parent
  * @param prevNotificationPreference The previous notification preference for the child report
  */
-function toggleSubscribeToChildReport(childReportID = '-1', parentReportAction: Partial<ReportAction> = {}, parentReportID = '-1', prevNotificationPreference?: NotificationPreference) {
-    if (childReportID !== '-1') {
+function toggleSubscribeToChildReport(childReportID = '', parentReportAction: Partial<ReportAction> = {}, parentReportID = '', prevNotificationPreference?: NotificationPreference) {
+    if (childReportID !== '') {
         openReport(childReportID);
-        const parentReportActionID = parentReportAction?.reportActionID ?? '-1';
+        // updateNotificationPreference cannot default parentReportActionID to '',
+        // otherwise it's behaviour will change
+        // eslint-disable-next-line rulesdir/no-default-id-values
+        const parentReportActionID = parentReportAction?.reportActionID ?? '';
```

Comment:
Please, follow rules described in the guide: https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#default-value-for-inexistent-ids

ID should not be and empty string